PyU4V/provisioning.py

```diff
@@ -1617,6 +1614,9 @@ def create_storage_group(
         if _async:
             payload.update(ASYNC_UPDATE)
 
+        if enable_mobility_id:
```

Comment:
I think you may have picked me up wrong here.  It was the parameters in def create_storage_group I was referring to.

Reply:
Ignore me, I had a look at the code.  enable_mobility_id is at the end everywhere